packages/oui-radio-group/src/index.spec.js

```diff
@@ -0,0 +1,85 @@
+describe("ouiRadioGroup", () => {
+    let TestUtils;
+    let $timeout;
+
+    beforeEach(angular.mock.module("oui.radio"));
+    beforeEach(angular.mock.module("oui.radio-group"));
+    beforeEach(angular.mock.module("oui.test-utils"));
+
+    beforeEach(inject((_TestUtils_, _$timeout_) => {
+        TestUtils = _TestUtils_;
+        $timeout = _$timeout_;
+    }));
+
+    const getRadioInputElement = (element) => angular.element(element[0].querySelector("input[type=radio]"));
+    const getRadioInputElementWithValue = (element, value) => angular.element(element[0].querySelector(`input[type=radio][value=${value}]`));
+    const getRadioGroupElement = (element) => angular.element(element[0]);
+
+    describe("Component", () => {
+        describe("attributes", () => {
+
+            it("should assign defined ame to child radios when name attribute is defined", () => {
+                const name = "foo";
+
+                const element = TestUtils.compileTemplate(`<oui-radio-group name=${name}><oui-radio></oui-radio></oui-radio-group>`);
+
+                expect(getRadioInputElement(element).prop("name")).toBe(name);
+            });
+
+            it("should assign a generated name to child radios  when name attribute is undefined ", () => {
+                const element = TestUtils.compileTemplate("<oui-radio-group><oui-radio></oui-radio></oui-radio-group>");
+
+                expect(getRadioInputElement(element).prop("name")).toMatch(/^oui-radio-group-\d+$/);
+            });
+
+
+            it("should add radiogroup role", () => {
+                const element = TestUtils.compileTemplate("<oui-radio-group></oui-radio-group>");
+                $timeout.flush();
+
+                expect(getRadioGroupElement(element).attr("role")).toEqual("radiogroup");
+            });
+        });
+
+        describe("on radio group model change", () => {
+            it("should update child radio models", () => {
```

Comment:
You should also test when you change the `ctrl.defaultValue` later. Now you are only testing the first initialisation of the component, but what happen if the model change later ?

```javascript
const ctrl = TestUtils.getController(element);
ctrl.defaultValue = "aValue";
$rootScope.$digest();

expect(..."aValue"...).toEqual(true);
expect(..."bValue"...).toEqual(false);
```